vpr/src/pack/cluster.cpp

```diff
@@ -1233,6 +1243,24 @@ static enum e_block_pack_status try_pack_molecule(t_cluster_placement_stats* clu
         return BLK_FAILED_FEASIBLE;
     }
 
+    //for loop to check if cluster PartitionRegion intersects with atom PartitionRegion
+    for (j = 0; j < molecule_size; j++) {
```

Comment:
int i_mol = 0, better style

Reply:
need a temporary cluster_pr that you pass to the intersect routine. like the convention